src/scripts/main.jsx

```diff
@@ -1,5 +1,6 @@
 // This puts a <style> tag in <head> of page with transpiled css.
 import '../styles/main.scss'
+import './googleAnalytics.js'
```

Comment:
Move googleAnalytics to the very top. Should be the first thing to load I think?